src/app/projects/[slug]/page.tsx

```
@@ -0,0 +1,41 @@
+}
+
+export default async function ProjectDetailPage({ params }: { params: ParamsPromise }) {
+  const { slug } = await params;
+  const project = projects.find(p => p.slug === slug);
+  if (!project) notFound();
+
+  const jsonLd = JSON.stringify(projectStructuredData(project));
+
+  return (
+    <>
+      <script type="application/ld+json" dangerouslySetInnerHTML={{ __html: jsonLd }} />
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)